boa_engine/src/vm/opcode/mod.rs

```diff
@@ -1326,6 +1323,20 @@ generate_impl! {
         /// Stack: **=>**
         Return,
 
+        /// Close an async generator function.
+        ///
+        /// Operands:
+        ///
+        /// Stack: **=>**
+        AsyncGeneratorClose,
+
+        /// Creates the generator generator object and yields.
```

Comment:
Same as boa_engine/src/vm/opcode/generator/mod.rs

```suggestion
        /// Creates the generator object and yields.
```